web/src/components/Header.js

```diff
@@ -99,15 +90,14 @@ class Header extends Component {
                         </Link>
                       </li>
                       <li>
-                        <Button
-                          size="small"
```

Comment:
Oooh, this is probably what was janking it up to begin with (which gives me some peace-of-mind about the text size on our other buttons).